database/DatabaseExampleSwiftUI/DatabaseExample/Shared/Models/CommentViewModel.swift

```diff
@@ -14,20 +14,30 @@
 //  limitations under the License.
 //
 
-import UIKit
+import SwiftUI
+import Firebase
 
-struct Comment {
+class CommentViewModel: ObservableObject, Identifiable {
```

Comment:
Do we expect the comments to be intractable or do we need to handle any events from the CommentView in the future? If no, then it doesn't look like we need a `CommentViewModel`. We probably need just a struct `Comment` instead.